google/cloud/spanner_v1/backup.py

```diff
@@ -280,16 +280,24 @@ def create(self):
         if (
             (
                 self._encryption_config
-                and self._encryption_config.kms_key_name
+                and (
```

Comment:
@hwin16 Admin features will directly use the autogenerated code and will not need any handwritten logic. So we can remove these changes in backup.py, database.py and test_database.py.